scripts/tests/basic_tests.bats

```diff
@@ -83,18 +96,23 @@ setup() {
   assert_equal "$str1_statom_diff" '1000'
 }
 
-@test "liquid stake IBCs automatically" {
-  # get module address 
-  ibcaddr=$($STR1_EXEC q stakeibc module-address stakeibc | awk '{print $NF}') 
-  module_atom=$($STR1_EXEC q bank balances $ibcaddr --denom $IBCATOM | GETBAL)
-  assert_equal "$module_atom" '1000'
-}
-
 # add test to register host zone 
 @test "host zone successfully registered" {
   run $STR1_EXEC q stakeibc show-host-zone GAIA
-  host_zone_info=$($STR1_EXEC q stakeibc get-host-zone  | awk '{print $NF}')
-  assert_line '  BaseDenom: ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9'
+  assert_line '  BaseDenom: uatom'
+  assert_line '  chainId: GAIA'
+  assert_line '    address: cosmos19l6d3d7k2pel8epgcpxc9np6fsvjpaaa06nm65vagwxap0e4jezq05mmvu'
+}
+
+# add test to see if assets are properly being staked on host zone
+@test "tokens staking on host zone" {
+  # run below test once ICQ is deployed
+  # VAL_ADDR='cosmosvaloper19e7sugzt8zaamk2wyydzgmg9n3ysylg6na6k6e'
+  # $GAIA1_EXEC q staking delegation cosmos19l6d3d7k2pel8epgcpxc9np6fsvjpaaa06nm65vagwxap0e4jezq05mmvu cosmosvaloper19e7sugzt8zaamk2wyydzgmg9n3ysylg6na6k6e
+  #   amount: "240"
+  del_balance_atom=$($STR1_EXEC q bank balances $DELEGATE_ADDR --denom uatom | GETBAL)
```

Comment:
Loving this testing framework

Reply:
Thank you! Should hopefully be easy enough to customize / add more detailed tests